tests/storage/test_store.py

```diff
@@ -149,19 +133,20 @@ def versioned_zarr_directory_store(tmp_path):
     "versioned_zarr_memory_store",
     "versioned_zarr_directory_store",
 ]
-store_names_is_versioned = [(name, "versioned" in name) for name in store_names]
 
 
+#------------------------------
 # TESTS
+#------------------------------
 
 
 # To use fixtures in parametrize, use the "request" fixture, as detailed here:
 # https://stackoverflow.com/questions/42014484/pytest-using-fixtures-as-arguments-in-parametrize
-@pytest.mark.parametrize("store_name, is_versioned", store_names_is_versioned)
-def test_insert_creates(store_name, is_versioned, request, ds1):
+@pytest.mark.parametrize("store_name", store_names)
+def test_insert_creates(store_name, request, ds1):
     store = request.getfixturevalue(store_name)
```

Comment:
A short doc-string comment on what this test is checking would be useful here (check that an empty store (not created yet?) registers as False until data is added?)

Same comment for the next 2 tests as well.